src/tools/python/classes/README.md

```diff
@@ -44,6 +44,6 @@ bob.sing()         # these are called instance methods, since they run on
                    # the instance of the class
 ```
 
-In the next couple of sub lessons we will look at each of these parts in more depth
+In the next couple of sub lessons we will look at each of these parts in more depth. 
 
-{{#authors lmaxwell24}}
\ No newline at end of file
+{{#authors lmaxwell24, stao5}}
```

Comment:
author annotations cannot have a space, it shows up weird
![image](https://github.com/user-attachments/assets/6ba4bb3c-8ec9-484f-8de4-fade9aa62cdc)